src/node.h

```diff
@@ -92,6 +92,7 @@ typedef struct node_t {
 
 	struct timeval udp_reply_sent;          /* Last time a (gratuitous) UDP probe reply was sent */
 	struct timeval udp_ping_sent;           /* Last time a UDP probe was sent */
+	long int udp_ping_rtt;                  /* Round trip time of UDP ping (in microseconds) */
```

Comment:
Actually, this can just be a regular `int`. 32 bits is already enough to handle RTTs of 35 minutes, by which time we've already sent another probe or have given up on the node altogether.